pkg/recipes/terraform/module.go

```diff
@@ -20,17 +20,14 @@ import (
 	"context"
 	"fmt"
 	"path/filepath"
+	"strings"
 
 	"github.com/hashicorp/terraform-config-inspect/tfconfig"
 	"github.com/hashicorp/terraform-exec/tfexec"
 	"github.com/radius-project/radius/pkg/recipes"
 	"github.com/radius-project/radius/pkg/recipes/recipecontext"
 )
 
-const (
-	moduleRootDir = ".terraform/modules"
-)
-
```

Comment:
why was this removed?

Reply:
`moduleRootDir` will need to change depending on whether there's a submodule or not, so this value was moved into the `inspectModules` function